source/backend/api/Helpers/Swagger/ConfigureSwaggerForOpenApi.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Globalization;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+using Swashbuckle.AspNetCore.SwaggerUI;
+
+namespace Pims.Api.Helpers.Swagger
```

Comment:
This class allows the config from appsettings to control the display of some fields in swagger.